CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 -  Added theme support for `UserMenuHeaderComponent`. 
 -  Red text highlight color for invalid form fields.
+-  Added theme support for HTML element `<input>`.
```

Comment:
I'm against trying to theme non-material elements with colors, I'd rephrase it.

Something like:

"Applied Open Sans font family to HTML `<input>` elements."

When we say we added theme support, I'm imagining full color, background color, typography for light/dark themes, which we don't have & I really want to avoid doing.